src/WindowManager.vala

```diff
@@ -472,7 +472,11 @@ namespace Gala {
                 neighbor.activate (display.get_current_time ());
             } else {
                 // if we didnt switch, show a nudge-over animation if one is not already in progress
-                play_nudge_animation (direction);
+                if (workspace_view.is_opened () && workspace_view is MultitaskingView) {
+                    (workspace_view as MultitaskingView).play_nudge_animation (direction);
```

Comment:
Causes warning 
```
../src/WindowManager.vala:476.21-476.77: warning: Access to possible `null'. Perform a check or use an unsafe cast.
```

The elementary recommended way of casting to avoid the warning is `((MultitaskingView) workspace_view).play_nudge_animation (direction)`.

Reply:
Fixed, thanks!!